spring-kafka-docs/src/main/antora/modules/ROOT/pages/retrytopic/features.adoc

```diff
@@ -261,3 +261,41 @@ protected Consumer<DeadLetterPublishingRecovererFactory>
 
 It is recommended that you use the provided resolvers when constructing the custom instance.
 
+[[exc-based-custom-dlt-routing]]
+== Routing of messages to custom DLTs based on thrown exceptions
+
+Starting with version 3.2.0, it's possible to route messages to custom DLTs based on the type of the exception, which has been thrown during their processing.
+In order to do that, you need to specify the routing.
+Routing customization consists of the specification of the additional destinations.
+Destinations in turn consist of two settings: the `suffix` and `exceptions`.
+When the exception type specified in `exceptions` has been thrown, the DLT containing the `suffix` will be considered as the target topic for the message before the general purpose DLT is considered.
+Take a look at the examples:
```

Comment:
I prefer to have an official neutral language in docs where it is depersonalized.
So, no phrases like `you`, `me`, `take a look` etc.

But... That's my personal preferences and I know there are many like that throughout the doc.

Also: this need to me mentioned in the `whats-new.adoc`